vuepress/docs/next/docs/README.md

```diff
@@ -32,8 +32,8 @@ On the Entando Platform, Curate and Share means development teams can bundle and
 
 | Documentation | Tutorials        |
 | :------------ | :-------- |
-| [Bundle and Component Descriptors](./curate/bundle-details.md) | [Install Bundles from a Private Git Repository](../tutorials/curate/private-git-repo.md) |
-| [Filtering Bundles](./curate/bundle-filters.md) | [Install Bundle Plugins from a Private Image Repo](../tutorials/curate/private-images.md) |
+| [Bundle and Component Descriptors](./curate/bundle-details.md) | [Install Bundles from a Private Image Repository](../tutorials/curate/bundle-private-images.md) |
+| [Filtering Bundles](./curate/bundle-filters.md) | [Install Bundle Plugins from a Private Image Repo](../tutorials/curate/ms-private-images.md) |
```

Comment:
Repo -> Repository to match the previous line? And I think if it should registry since we have Docker Registries vs Git Repositories.